decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
```

Comment:
- Hey, we also need to access the nameable by adding the `attr_accessor :nameable_obj`. please, kindly add it to enable the read and write access.